frontend/src/Search/Search.js

```diff
@@ -12,7 +12,21 @@ const StyledDiv = styled.div`
 
 const Search = React.memo(({ history, restaurants, updateBounds }) => {
   // will need to fix in the future for resizing window
-  const shouldRenderMap = useMemo(() => window.innerWidth > 900, []);
+  const [shouldRenderMap, setShouldRenderMap] = useState(
+    window.innerWidth > 900
```

Comment:
It's probably worth making 900 a constant and placing it in utils or something. That way if you ever want to update the minimum width globally, you just have to dial one variable and everything will adjust.